src/universal/containers/app/SignUpContainer/SignUpContainer.mobile.tsx

```diff
@@ -0,0 +1,145 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import * as React from 'react';
+import { 
+  RouteComponentProps,
+  Redirect, 
+  withRouter,
+} from 'react-router';
+
+import { ConnectedReduxProps } from '@universal/state/configureStore';
+import { requestSignUpUser } from '@actions/userActions';
+import SignUp from '@components/app/SignUp/SignUp.mobile';
+import Terms from '@components/app/Terms/Terms.mobile';
+
+class SignUpContainer extends React.Component<SignUpContainerProps, SignUpContainerStates> {
+  constructor(props) {
+    super(props);
+    this.handleChangeEmail = this.handleChangeEmail.bind(this);
+    this.handleChangePassword = this.handleChangePassword.bind(this);
+    this.handleChangePasswordDoubleCheck = this.handleChangePasswordDoubleCheck.bind(this);
+    this.handleChangeUsername = this.handleChangeUsername.bind(this);
+    this.handleClickSignUp = this.handleClickSignUp.bind(this);
+    this.handleClickTermsAgree = this.handleClickTermsAgree.bind(this);
+    this.state = {
+      email: '',
+      password: '',
+      passwordDoubleCheck: '',
```

Comment:
maybe we could name these as `password1`, `password2`. What do you think? @UnoRich